components/sidebar_right/sidebar_right.tsx

```diff
@@ -159,9 +165,28 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
             this.props.actions.setRhsExpanded(false);
         }
 
+        // close when changing teams, close when changing productsRHS UX
```

Comment:
Is "RHS UX" meant to be at the end of this comment?

Reply:
typo